### DISCUSSION TITLE
Form Payload Issue? (Cloud Editor)

### DISCUSSION THREAD

**Discussion by VikingDadMedic** [Budibase/budibase#3461] [Help]
What am I missing when it comes to multi step forms - why is half the form an array? Tried removing steps, creating different input types, etc. Always happens... I am really hoping this is user error on my part and not just buggy issues.

![Screen Shot 2021-11-20 at 12 52 59 PM](https://user-images.githubusercontent.com/91684138/142736420-475d94c4-54b9-4656-ba0d-b3a8bc5da9e6.png)

**Comment by mjashanks** [comment]
Hey Ryan. Apologies, I do not understand what issue you are having. What do you mean by "Half the form is an array?" - I can't see any arrays in your screenshot.

Thanks.
